src/components/idleMembers/card/index.tsx

```diff
@@ -1,18 +1,29 @@
 import { FC, SyntheticEvent, useState } from 'react';
 import Image from 'next/image';
 import classNames from '@/components/idleMembers/card/card.module.scss';
-import { DUMMY_PROFILE } from '@/components/constants/display-sections.js';
+import { DUMMY_PROFILE, DUMMY_PROFILE_PATH } from '@/components/constants/display-sections.js';
+import { getCloudinaryImgURL } from '@/helperFunctions/getCloudinaryImageUrl';
 
-const IMAGE_URL = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
+const CLOUDINARY_IMAGE_CONFIGS = 'w_160,h_160'
 
 type Props = {
-  idleMemberUserName: string
+  idleMemberUserName: string,
+  imageURL:string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
   const [isImageAvailable, setIsImageAvailable] = useState(false);
 
-  const assigneeProfilePic = (name: string) => `${IMAGE_URL}/${name}/img.png`;
+  const userImageUrl = `/dummyProfile.png`
```

Comment:
Why does this not use the `DUMMY_PROFILE_PATH` constant?

Reply:
Yeah, should have used the constant. Taken care of in the latest commit. Thanks